pkg/context/context.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package context
+
+import (
+	"github.com/spf13/viper"
+	"github.com/trustbloc/sidetree-core-go/pkg/api/protocol"
+	"github.com/trustbloc/sidetree-core-go/pkg/batch"
+	"github.com/trustbloc/sidetree-core-go/pkg/processor"
+
+	"github.com/trustbloc/sidetree-core-go/pkg/mocks"
+
+	servermocks "github.com/trustbloc/sidetree-node/pkg/mocks"
+)
+
+func New(cfg *viper.Viper) (*ServerContext, error) { // nolint
+
+	opsStore := mocks.NewMockOperationStore(nil)
+	cas := servermocks.NewMockCasClient(nil)
+	cas.OpsStore = opsStore
+
+	ctx := &ServerContext{
+		ProtocolClient:   servermocks.NewMockProtocolClient(),
+		CasClient:        cas,
+		BlockchainClient: mocks.NewMockBlockchainClient(nil),
+		OperationStoreClient: opsStore,
+	}
+
+	return ctx, nil
+
+}
+
+// ServerContext implements batch context
+type ServerContext struct {
+	ProtocolClient   *servermocks.MockProtocolClient
+	CasClient        *servermocks.MockCasClient
+	BlockchainClient *mocks.MockBlockchainClient
+	OperationStoreClient *mocks.MockOperationStore
+}
```

Comment:
Has go fmt applied to this file ? Doesn't look like this is properly formatted/indented.

Reply:
.golangci.yml was missing, fixed